README.md

```diff
@@ -1,17 +1,57 @@
-# SpatialOS Unreal GDK Test Suite
+# SpatialOS GDK Test Suite
+LICENSE: Use of the contents of this repository is subject to the [Improbable license](LICENSE.md)
+REPOSITORY: github.com/spatialos/UnrealGDKTestSuite
```

Comment:
`REPOSITORY: github.com/spatialos/UnrealGDKTestSuite`
Should probably be a hyperlink no?